Tools/jit/_stencils.py

```diff
@@ -364,7 +364,7 @@ def _get_trampoline_mask(self) -> str:
 
     def as_c(self, opname: str) -> str:
         """Dump this hole as a StencilGroup initializer."""
-        return f"{{emit_{opname}, {len(self.code.body)}, {len(self.data.body)}, {self._get_trampoline_mask()}}}"
+        return f"{{emit_{opname}, {len(self.code.body)}, {self._get_trampoline_mask()}, {len(self.data.body)}}}"
```

Comment:
Can you revert this change? I found it a bit clearer before.

Reply:
Done.